actions/test/run_local.ts

```diff
@@ -25,9 +25,24 @@ const main = async () => {
   // Get provider
   const provider = await getProvider(testRuntime.context, chainId);
 
-  // Run one of the 2 Execution modes (single block, or watch mode)
+  // Run one of the 3 Execution modes:
+  //  - Single tx: Process just one tx
+  //  - Single block: Process just one block
+  //  - Watch mode: Watch for new blocks, and process them
+  const txEnv = process.env.TX;
   const blockNumberEnv = process.env.BLOCK_NUMBER;
-  if (blockNumberEnv) {
+  if (txEnv) {
```

Comment:
Maybe mention in the docs/readme the order of precedence for the env vars?

Reply:
done